### PR TITLE
summary update

### PR DESCRIPTION
**Pull request by jessegoodier**
1. Clean up summary
2. Fixes link on aws-cur-setup, spelling error

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -12,28 +12,30 @@
...
       * [Azure Rate Card Configuration](azure-config.md)
-    * [GCP Cloud Integration](gcp-out-of-cluster.md)
+    * [GCP Cloud Billing Integration](gcp-out-of-cluster.md)
       * [Creating a Google Service Account](google-service-account-thanos.md)
   * [Multi-Cluster](multi-cluster.md)
-    * [Federated ETL](federated-etl.md)
+    * [ETL Federation (preferred)](federated-etl.md)
       * [Backups and Alerting](federated-etl-backups-alerting.md)
-    * [Thanos Federation](thanos-setup.md)
+    * [Thanos Federation (fully supported)](thanos-setup.md)
```
Comment (chipzoller) [inline SUMMARY.md:27]: Why the "fully supported" qualification here? Seems a bit odd for a document/page title in a product's official documentation.

File: thanos-setup.md [hunk 2]
```diff
@@ -6,12 +6,16 @@ This feature is only officially supported on Kubecost Enterprise plans.
 
 There are two primary methods to aggregate all cluster information back to a single Kubecost UI described in [Multi-Cluster Kubecost](multi-cluster.md#enterprise-federation).
 
-Below is the configuration guide using Kubecost Thanos Federation.
+The *preferred* method for multi-cluster is [ETL Federation](federated-etl.md). The configuration guide below is for Kubecost Thanos Federation, which may not scale as well as ETL Federation in large environments.
```
Comment (jessegoodier) [inline thanos-setup.md:9]: @chipzoller Can you take a look at these changes?

File: thanos-setup.md [hunk 3]
```diff
@@ -6,12 +6,16 @@ This feature is only officially supported on Kubecost Enterprise plans.
...
 
 ## Configuration
 
-1. Follow steps [here](long-term-storage.md#option-b-out-of-cluster-storage-thanos) to enable Thanos durable storage on a Master cluster.
-2.  Repeat the process in Step 1 for each additional secondary cluster, with the following Thanos recommendations:
+Thanos is a tool to aggregate prometheus metrics to a central object storage (S3 compatible) bucket. Thanos is implemented as a sidecar on the Prometheus pod on all clusters.
```
Comment (chipzoller) [inline thanos-setup.md:13]: "Prometheus" is a proper noun.

File: thanos-setup.md [hunk 4]
```diff
@@ -6,12 +6,16 @@ This feature is only officially supported on Kubecost Enterprise plans.
...
+
+1. Follow steps [here](long-term-storage.md#option-b-out-of-cluster-storage-thanos) to enable all required Thanos components on a Kubecost Primary cluster, including the prometheus sidecar.
```
Comment (chipzoller) [inline thanos-setup.md:15]: "Prometheus"

Comment (chipzoller) [reply thanos-setup.md:15]: "primary"

**Review by chipzoller — CHANGES_REQUESTED** [review]

**Review by bstuder99 — APPROVED** [review]

**Review by chipzoller — APPROVED** [review]

**Pull request merged**